sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/internal/PathPatternPredicate.java

```diff
@@ -34,7 +35,7 @@
 
   @Override
   public boolean apply(InputFile f) {
-    return pattern.match(f.absolutePath(), f.relativePath());
+    return pattern.match(f.path(), Paths.get(f.relativePath()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "path"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "relativePath"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)